R/infrastructure.R

```diff
@@ -74,7 +74,7 @@ use_rstudio <- function(pkg = ".") {
 
   path <- file.path(pkg$path, paste0(pkg$package, ".Rproj"))
   if (file.exists(path)) {
-    stop(pkg$package, ".Rproj already exists", call. = FALSE)
```

Comment:
I think this should probably still print a `message()` - see the new behaviour for `use_git()` and `use_github()`